crates/aptos/src/move_tool/transactional_tests_runner.rs

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+/// Most of the code below comes from the crate `datatest-stable`. Because the limitation of `datatest-stable`,
+/// we are not able to run transactional tests as a subcommand of the Aptos CLI. Therefore, we need to duplicate code
+/// here and make minor modifications.
+///
+use clap::Parser;
+
+use std::{
+    io::{self, Write},
+    num::NonZeroUsize,
+    panic::{catch_unwind, AssertUnwindSafe},
+    path::{Path, PathBuf},
+    process,
+    sync::mpsc::{channel, Sender},
+    thread,
+};
+
+use termcolor::{Color, ColorChoice, ColorSpec, StandardStream, WriteColor};
+
+type Result<T> = std::result::Result<T, Box<dyn std::error::Error>>;
+
+/// Run Move transactional tests
+#[derive(Parser, Clone)]
+pub struct TransactionalTestOpts {
+    /// The FILTER string is tested against the name of all tests, and only those tests whose names
+    /// contain the filter are run.
+    #[clap(long)]
```

Comment:
do we want to provide both clap(long, short)? typing these option in full gives me CTS

Reply:
We're not doing shorts, given the number of commands we have it's likely to collide & get confusing.